### PR TITLE
Updated README to explictly tell how to proceed on fresh install

### PR DESCRIPTION
**Pull request by george-gca**
Updated README to explicitly tell how to proceed on creating your own version based on this template. Also removed the video tutorial since it was for an older version, recommending to fork the repo and it was missing the `--lsi` parameter from jekyll command. Also was not using docker, the current recommended way.

### CODE REVIEW

**Review by pourmand1376 — APPROVED** [review]
Great. 

I think we should also add some more information about docker. The tutorial is not complete. 

For example, we should mention they should use `localhost:8888` on browser not `0.0.0.0:8888` which shows on terminal.

**Comment by george-gca** [comment]
What else other than the ip you think is missing @pourmand1376?

File: README.md [hunk 1]
```diff
@@ -210,16 +211,18 @@ Why Jekyll? Read [Andrej Karpathy's blog post](https://karpathy.github.io/2014/0
...
-note that the name of your repository must be `<your-github-username>.github.io` or `<your-github-orgname>.github.io`, as stated in the [GitHub pages docs](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites).
-For more information on how to deploy your site, check the [Deployment](#deployment) section below. After you created your new repository, just download it to your machine:
+1. Create a new repository using this template. For this, click on `Use this template -> Create a new repository` above the file list. If you plan to upload your site to `<your-github-username>.github.io`, note that the name of your repository MUST BE `<your-github-username>.github.io` or `<your-github-orgname>.github.io`, as stated in the [GitHub pages docs](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites).
+2. In this new repository, go to `Settings -> Actions -> General -> Workflow permissions` and give `Read and write permissions` to GitHub Actions.
+3. Open file `_config.yml`, set `url` to `https://<your-github-username>.github.io` and leave `baseurl` **empty**.
```
Comment (sethajacobson) [inline README.md:218]: This change needs to be committed, too.

Comment (george-gca) [reply README.md:218]: Which change exactly?

**Comment by george-gca** [comment]
Added missing info from #1781

**Pull request merged**